### ISSUE TITLE
Tweak peek inside notebooks

### ISSUE THREAD

**Issue by jrieken** [microsoft/vscode#101513]
@isidorn had shared this with me a while back

* have a notebook
* run reference search or F8 for errors
<img width="913" alt="Screenshot 2020-07-01 at 17 04 56" src="https://user-images.githubusercontent.com/1794099/86259989-ffea1c80-bbbc-11ea-8357-81be4fbd07c9.png">

Issues/suggestion: 

* there are many lines for peek and for the focus border
* the peek editor has some whitespace at its left and right side
* instead of lines we should experiment with more background color, e,g drop the outline and only use background

**Comment by isidorn** [comment]
Yeah, my biggest problem is too many lines, borders and rectangles inside rectangles.
Simply the peek UI was designed for the editor and it does not seem fitting for a notebook cell.

Agree with @jrieken that a first step would be to try to drop some lines and experiment more with background
I do believe that regular Editors can also benefit from this, as I think this UI works best when there is exactly one editor, but when there are editors side by side it can be simplified.
